crates/core_arch/src/aarch64/neon/mod.rs

```diff
@@ -795,6 +796,292 @@ pub unsafe fn vld1q_f64(ptr: *const f64) -> float64x2_t {
+        &a as *const uint64x1_t as *const u64,
+        ptr as *mut u64,
+        size_of::<uint64x1_t>(),
+    )
+}
+
+/// Store multiple single-element structures from one, two, three, or four registers.
+#[inline]
+#[target_feature(enable = "neon")]
+#[cfg_attr(test, assert_instr(str))]
+#[allow(clippy::cast_ptr_alignment)]
+pub unsafe fn vst1q_u64(ptr: *mut u64, a: uint64x2_t) {
+    copy_nonoverlapping(
+        &a as *const uint64x2_t as *const u64,
+        ptr as *mut u64,
+        size_of::<uint64x2_t>(),
+    )
+}
+
+/// Store multiple single-element structures from one, two, three, or four registers.
+#[inline]
+#[target_feature(enable = "neon")]
+#[cfg_attr(test, assert_instr(str))]
+#[allow(clippy::cast_ptr_alignment)]
+pub unsafe fn vst1_p8(ptr: *mut p8, a: poly8x8_t) {
+    copy_nonoverlapping(
+        &a as *const poly8x8_t as *const p8,
+        ptr as *mut p8,
+        size_of::<poly8x8_t>(),
+    )
+}
+
+/// Store multiple single-element structures from one, two, three, or four registers.
+#[inline]
+#[target_feature(enable = "neon")]
+#[cfg_attr(test, assert_instr(str))]
+#[allow(clippy::cast_ptr_alignment)]
+pub unsafe fn vst1q_p8(ptr: *mut p8, a: poly8x16_t) {
+    copy_nonoverlapping(
+        &a as *const poly8x16_t as *const p8,
+        ptr as *mut p8,
+        size_of::<poly8x16_t>(),
+    )
+}
+
+/// Store multiple single-element structures from one, two, three, or four registers.
+#[inline]
+#[target_feature(enable = "neon")]
+#[cfg_attr(test, assert_instr(str))]
+#[allow(clippy::cast_ptr_alignment)]
+pub unsafe fn vst1_p16(ptr: *mut p16, a: poly16x4_t) {
+    copy_nonoverlapping(
+        &a as *const poly16x4_t as *const p16,
+        ptr as *mut p16,
+        size_of::<poly16x4_t>(),
+    )
+}
+
+/// Store multiple single-element structures from one, two, three, or four registers.
+#[inline]
+#[target_feature(enable = "neon")]
+#[cfg_attr(test, assert_instr(str))]
+#[allow(clippy::cast_ptr_alignment)]
+pub unsafe fn vst1q_p16(ptr: *mut p16, a: poly16x8_t) {
+    copy_nonoverlapping(
+        &a as *const poly16x8_t as *const p16,
+        ptr as *mut p16,
+        size_of::<poly16x8_t>(),
+    )
+}
+
+// Store multiple single-element structures from one, two, three, or four registers.
+#[inline]
+#[target_feature(enable = "neon")]
+#[cfg_attr(test, assert_instr(str))]
+#[allow(clippy::cast_ptr_alignment)]
+pub unsafe fn vst1_f32(ptr: *mut f32, a: float32x2_t) {
+    copy_nonoverlapping(
+        &a as *const float32x2_t as *const f32,
+        ptr as *mut f32,
+        size_of::<float32x2_t>(),
+    )
+}
+
+// Store multiple single-element structures from one, two, three, or four registers.
+#[inline]
+#[target_feature(enable = "neon")]
+#[cfg_attr(test, assert_instr(str))]
+#[allow(clippy::cast_ptr_alignment)]
+pub unsafe fn vst1q_f32(ptr: *mut f32, a: float32x4_t) {
+    copy_nonoverlapping(
+        &a as *const float32x4_t as *const f32,
+        ptr as *mut f32,
+        size_of::<float32x4_t>(),
+    )
+}
+
```

Comment:
Can you add `vst1_f64` and `vst1_p64` instructions at the same time?:)

Reply:
Oh snap, I forgot those? Sure, I can add them too.

Reply:
I added the missing intrinsics for aarch64.